generators/app/templates/src/main/webapp/angular/app/admin/entity-audit/_entity-audit-modal.component.css

```diff
@@ -0,0 +1,20 @@
+/* NOTE: for now the /deep/ shadow-piercing descendant combinator is
+ * required because Angular defaults to emulated view encapsulation and
+ * preprocesses all component styles to approximate shadow scoping
+ * rules. This means these styles wouldn't apply to the HTML generated
+ * by ng-diff-match-patch.
+ *
+ * This shouldn't be required when browsers support native
+ * encapsulation, at which point /deep/ will also be deprecated/removed
+ * see https://angular.io/guide/component-styles
+ */
+
+:host /deep/ ins {
```

Comment:
probably better to inline the style in the compoenent so that you dont have to handle sass and stuff